README.md

```diff
@@ -1,16 +1,18 @@
-> Because I'm no longer using Windows frequently, I'm currently looking for an additional maintainer to assist in the Winfetch development. [#17](https://github.com/lptstr/winfetch/issues/17)
-<br>
+# Posh Winfetch
+> A command-line system information utility for Windows 10
 
 <h3 align="center"><img src="https://lptstr.github.io/lptstr-images/proj/winfetch/logo.png" alt="logo" height="100px"></h3>
-<p align="center">A command-line system information utility for Windows 10</p>
```

Comment:
This line should be kept (see my comment above).

Reply:
OK